pkgs/by-name/ca/canaille/package.nix

```
@@ -0,0 +1,133 @@
+{
+  lib,
+  python3,
+  fetchFromGitLab,
+  openldap,
+  nixosTests,
+}:
+
+let
+  python = python3.override {
+    packageOverrides = final: prev: {
+      sentry-sdk = prev.sentry-sdk.overridePythonAttrs (old: {
+        dependencies = old.dependencies ++ prev.sentry-sdk.optional-dependencies.flask;
+      });
+      # coverage = prev.coverage.overridePythonAttrs (old: {
+      #   dependencies = (old.dependencies or []) ++ prev.coverage.optional-dependencies.toml;
+      # });
+    };
+  };
+in
+python.pkgs.buildPythonApplication rec {
+  pname = "canaille";
+  version = "0.0.54";
+  pyproject = true;
+
+  disabled = python.pkgs.pythonOlder "3.9";
+
+  src = fetchFromGitLab {
+    owner = "yaal";
+    repo = "canaille";
+    rev = version;
+    sha256 = "sha256-MowvgZsb4i0hsqQbOsPl0dbLxxW0J1KAqAz2xEQ+Yks=";
+  };
+
+  # See https://github.com/NixOS/nixpkgs/issues/103325
+  postPatch = ''
+    substituteInPlace pyproject.toml \
+      --replace "poetry>=1.0.0" "poetry-core" \
+      --replace "poetry.masonry.api" "poetry.core.masonry.api"
```

Can you submit this upstream?
```suggestion
    substituteInPlace pyproject.toml \
      --replace-fail "poetry>=1.0.0" "poetry-core" \
      --replace-fail "poetry.masonry.api" "poetry.core.masonry.api"
```

---

Upstream PR: https://gitlab.com/yaal/canaille/-/merge_requests/178